.github/workflows/robotmk_build.yaml

```diff
@@ -10,8 +10,6 @@ jobs:
       - uses: actions/checkout@v4
```

Comment:
You wrote "warnins" in the commit message.

Reply:
Done